ranger/gui/ui.py

```diff
@@ -302,10 +302,12 @@ def setup(self):
         self.taskview.visible = False
         self.add_child(self.taskview)
 
-        # Create the status bar
+        # Create the status bar whether the setting is "True" or "False"
```

Comment:
This means it doesn't matter whether the setting is true or false.

Reply:
Done